js/src/components/customer-effort-score-prompt/index.js

```diff
@@ -47,6 +48,8 @@ import localStorage from '.~/utils/localStorage';
  * @return {JSX.Element} Rendered element.
  */
 const CustomerEffortScorePrompt = ( { eventContext, label } ) => {
+	useEffectRemoveNotice( label, 'core/notices2' );
```

Comment:
💅 
Looks like the store key `core/notices2` might be [changed](https://github.com/woocommerce/woocommerce/blob/6.6.0/packages/js/notices/src/store/index.js#L13-L14) or be [removed](https://github.com/woocommerce/woocommerce/tree/6.6.0/packages/js/notices#notices) from WC someday. Perhaps we could add a code comment here for this.